bigtable/tests/data_integration_test.cc

```diff
@@ -28,8 +28,15 @@ class DataIntegrationTest : public bigtable::testing::TableIntegrationTest {
                  std::vector<bigtable::Cell> const& cells);
 
   std::string const family = "family";
-  bigtable::TableConfig table_config = bigtable::TableConfig(
-      {{family, bigtable::GcRule::MaxNumVersions(10)}}, {});
+  std::string const family1 = "AmazingFamily";
```

Comment:
Try to use neutral column family names please.  "fam1", "fam2", "fam3" seem Okay.